components/pricing_modal/pricing_modal.scss

```diff
@@ -298,7 +298,7 @@
                     position: relative;
                     height: auto;
                     padding: 37px 24px 24px 24px;
-                    background-color: var(--title-color-indigo-500);
+                    background-color: var(--sidebar-bg);
```

Comment:
Wondering about this change, as the design specifies the background color not changing with themes:
https://www.figma.com/file/rWB5bbi3DvpmVwVMc2xfOH/Update-Plans?node-id=0%3A1&t=EfrG35qAEXy6HlnJ-0

<img width="1014" alt="Screen Shot 2022-11-23 at 11 08 13 AM" src="https://user-images.githubusercontent.com/79058848/203594871-c1d7f8a6-84f5-40fd-9a9b-0f6708ef20b2.png">

cc @AGMETEOR @annelieseklein

Reply:
@julmondragon I will revert this we have a separate ticket for some colour issues related to this modal.

Reply:
https://mattermost.atlassian.net/browse/MM-48570

Reply:
Thanks @AshishDhama !